playground/src/App.svelte

```diff
@@ -13,6 +14,10 @@
   })().catch((error) =>
     console.error("Error in setting up the filesystem:", error),
   );
+
+  $effect(() => {
+    init();
+  });
```

Comment:
the effect here is pointless it can just be at the setupPromise function

Reply:
But it is a different thing...

Reply:
what do you mean?

Reply:
one function is setting up the fs, the other one initializing the web assembly, why putting it together?

Reply:
because you dont want the app to be rendered before finishing the init, also there is no point in the effect anyway because the script tag is only ran once.